vip-dashboard/components/widget-contact.php

```diff
@@ -44,8 +44,11 @@ function render_vip_dashboard_widget_contact() {
 					<label for="contact-form__details">Details</label>
 				</div>
 				<div class="contact-form__input">
-					<textarea name="details" rows="4" id="contact-form__details"
-							placeholder="Please be descriptive"></textarea>
+					<textarea name="details" rows="8" id="contact-form__details" placeholder="Please include as much information possible about the issue, including relevant domain names and your investigation so far."></textarea>
+					<strong>
+						Note: please read our <a href="https://wpvip.com/documentation/general-ticket-guidelines/" target="_blank" rel="noopener">full ticket guidelines</a>
```

Comment:
This URL is old, and redirects to `https://docs.wpvip.com/technical-references/vip-support/` - that URL should have been used.

Reply:
Opened https://github.com/Automattic/vip-go-mu-plugins/pull/3548